packages/styles/src/tokens/colorsPalette.ts

```diff
@@ -0,0 +1,246 @@
+  green_100: "#DCFCE7",
+  green_200: "#BBF7D0",
+  green_300: "#86EFAC",
+  green_400: "#4ADE80",
+  green_500: "#22C55E",
+  green_600: "#16A34A",
+  green_700: "#15803D",
+  green_800: "#166534",
+  green_900: "#14532D",
+  green_950: "#052E16",
+} as const;
+
+export const teal = {
+  teal_50: "#F0FDFA",
+  teal_100: "#CCFBF1",
+  teal_200: "#99F6E4",
+  teal_300: "#5EEAD4",
+  teal_400: "#2DD4BF",
+  teal_500: "#14B8A6",
+  teal_600: "#0D9488",
+  teal_700: "#0F766E",
+  teal_800: "#115E59",
+  teal_900: "#134E4A",
+  teal_950: "#042F2E",
+} as const;
+
+export const cyan = {
+  cyan_50: "#ECFEFF",
+  cyan_100: "#CFFAFE",
+  cyan_200: "#A5F3FC",
+  cyan_300: "#67E8F9",
+  cyan_400: "#22D3EE",
+  cyan_500: "#06B6D4",
+  cyan_600: "#0891B2",
+  cyan_700: "#0E7490",
+  cyan_800: "#155E75",
+  cyan_900: "#164E63",
+  cyan_950: "#083344",
+} as const;
+
+export const sky = {
+  sky_50: "#F0F9FF",
+  sky_100: "#E0F2FE",
+  sky_200: "#BAE6FD",
+  sky_300: "#7DD3FC",
+  sky_400: "#38BDF8",
+  sky_500: "#0EA5E9",
+  sky_600: "#0284C7",
+  sky_700: "#0369A1",
+  sky_800: "#075985",
+  sky_900: "#0C4A6E",
+  sky_950: "#082F49",
+} as const;
+
+export const indigo = {
+  indigo_50: "#EEF2FF",
+  indigo_100: "#E0E7FF",
+  indigo_200: "#C7D2FE",
+  indigo_300: "#A5B4FC",
+  indigo_400: "#818CF8",
+  indigo_500: "#6366F1",
+  indigo_600: "#4F46E5",
+  indigo_700: "#4338CA",
+  indigo_800: "#3730A3",
+  indigo_900: "#312E81",
+  indigo_950: "#1E1B4B",
+} as const;
+
+export const violet = {
+  violet_50: "#F5F3FF",
+  violet_100: "#EDE9FE",
+  violet_200: "#DDD6FE",
+  violet_300: "#C4B5FD",
+  violet_400: "#A78BFA",
+  violet_500: "#8B5CF6",
+  violet_600: "#7C3AED",
+  violet_700: "#6D28D9",
+  violet_800: "#5B21B6",
+  violet_900: "#4C1D95",
+  violet_950: "#2E1065",
+} as const;
+
+export const fuchsia = {
+  fuchsia_50: "#FDF4FF",
+  fuchsia_100: "#FAE8FF",
+  fuchsia_200: "#F5D0FE",
+  fuchsia_300: "#F0ABFC",
+  fuchsia_400: "#E879F9",
+  fuchsia_500: "#D946EF",
+  fuchsia_600: "#C026D3",
+  fuchsia_700: "#A21CAF",
+  fuchsia_800: "#86198F",
+  fuchsia_900: "#701A75",
+  fuchsia_950: "#4A044E",
+} as const;
+
+export const pink = {
+  pink_50: "#FDF2F8",
+  pink_100: "#FCE7F3",
+  pink_200: "#FBCFE8",
+  pink_300: "#F9A8D4",
+  pink_400: "#F472B6",
+  pink_500: "#EC4899",
+  pink_600: "#DB2777",
+  pink_700: "#BE185D",
+  pink_800: "#9D174D",
+  pink_900: "#831843",
+  pink_950: "#500724",
+} as const;
+
+export const rose = {
+  rose_50: "#FFF1F2",
+  rose_100: "#FFE4E6",
+  rose_200: "#FECDD3",
+  rose_300: "#FDA4AF",
+  rose_400: "#FB7185",
+  rose_500: "#F43F5E",
+  rose_600: "#E11D48",
+  rose_700: "#BE123C",
+  rose_800: "#9F1239",
+  rose_900: "#881337",
+  rose_950: "#4C0519",
+} as const;
+
+/**
+ * Base color palette tokens. Should rarely be used directly (eg. charts)
+ * @private WIP. Currently in used for Pentaho+ theme
+ */
+export default {
+  ...slate,
+  ...neutral,
+  ...blue,
+  ...red,
+  ...orange,
+  ...yellow,
```

Comment:
Instead of spreading all the tokens in a giant flat object, we could nest the colors for improved discoverability.

Flat object:

<img width="207" alt="image" src="https://github.com/lumada-design/hv-uikit-react/assets/638946/fc876a10-c2b1-43c5-a3ac-34b00459d56e">

Nested object:

<img width="188" alt="image" src="https://github.com/lumada-design/hv-uikit-react/assets/638946/652f1049-597b-4f2e-a9fd-256af16f0cba">
<img width="187" alt="image" src="https://github.com/lumada-design/hv-uikit-react/assets/638946/213f0ad4-c9a0-484e-8d8d-d7a3a7aaf9b6">

Reply:
I prefer the nested object too, it looks at lot more "clean" and easier to use.

Reply:
I agree, it makes much more sense to me.